config/jobs/periodic/docker-in-docker/periodic-all-in-one.yaml

```diff
@@ -0,0 +1,29 @@
+periodics:
```

Comment:
instead of a periodic job, can this be a presubmit job that runs against any 20-* branch in docker-ce-packaging?

Kind of the idea but probably not quite right (and still has some bit from https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md):

presubmits:
  docker/docker-ce-packaging:
  - name: docker-all-in-one-test
    decorate: true 
    always_run: true   
    skip_report: true  
    max_concurrency: 5
    spec: {your spec...}
    branches:             # Regexps, only run against these branches.
    - ^20
    trigger: "(?m)qux test this( please)?" # Regexp, see discussion.
    rerun_command: "qux test this please"  # String, see discussion.

Reply:
In order to do this, we need to have webhook registered into that repo and some write access to the repository to reflect the test results etc.. which doesn't look like a feasible idea atm, having that said we are working on a plan to run the presubmit/postsubmit jobs for the external owner repos,.. will keep you posted!

Reply:
Can I ask you more details about that plan ?

Reply:
It may take some more time as we are busy with other deliverables, keep one thing in mind that this may be additional permissions like PR status write access, PR comment access etc, so even it also depends on whether the project is comfortable giving those access to the bot.

Reply:
> we are working on a plan to run the presubmit/postsubmit jobs for the external owned repos

which repos? and who's `we`? the runtimes squad has been working with docker for over a year now so we definitely should stay in sync and not have two different groups of people working on this without communication at least. there's also the issue of the contract, etc.,

Reply:
> > we are working on a plan to run the presubmit/postsubmit jobs for the external owned repos
> 
> which repos? and who's `we`? the runtimes squad has been working with docker for over a year now so we definitely should stay in sync and not have two different groups of people working on this without communication at least. there's also the issue of the contract, etc., 

Nothing to do with docker, we are just working on building a solution to run the jobs external owned repository, nothing to do with any individual project.. it is up to different squads if they want to use that solution...

Reply:
ok great. and thanks. just an english-ism that sounded like you had specific repos targeted.